docs/components/board/_index.md

```diff
@@ -35,8 +35,8 @@ For model-specific configuration information, click on one of the following mode
 | ----- | ----------- |
 | [`pi`](pi/) | [Raspberry Pi 4](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/), [Raspberry Pi 3](https://www.raspberrypi.com/products/raspberry-pi-3-model-b/) or [Raspberry Pi Zero 2 W](https://www.raspberrypi.com/products/raspberry-pi-zero-2-w/) |
 | [`ti`](ti/) | [Texas Instruments TDA4VM](https://devices.amazonaws.com/detail/a3G8a00000E2QErEAN/TI-TDA4VM-Starter-Kit-for-Edge-AI-vision-systems) |
-| [`beaglebone`](beaglebone/) | [BeagleBoard's BeagleBone AI 64](https://beagleboard.org/ai-64) |
-| [`jetson`](jetson/) | [NVIDIA Jetson AGX Orin](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Xavier NX](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-agx-xavier/), [NVIDIA Jetson  Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-nano/) |
+| [`beaglebone`](beaglebone/) | [BeagleBoard's BeagleBone AI-64](https://beagleboard.org/ai-64) |
+| [`jetson`](jetson/) | [NVIDIA Jetson AGX Orin](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Orin Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Xavier NX](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-agx-xavier/), [NVIDIA Jetson Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-nano/) |
```

Comment:
:exploding_head: I wrote a comment saying to change "AGX Orin" to "Orin AGX," then went to double-check that. You've got it right here, and I've been saying it wrong for 9 months! I wonder if I've influenced any of the documentation that is now wrong...

Reply:
I got surprised that the AGX Orin and Orin Nano have the same URL, but having visited that website, yes, the URL is for both of them!

Reply:
The Jetson Xavier NX links to the Jetson AGX Xavier page. I wonder if https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-agx-xavier/ is supposed to be https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-xavier-nx/